modules/vaos/app/services/vaos/v2/mobile_facility_service.rb

```diff
@@ -6,14 +6,16 @@
 module VAOS
   module V2
     class MobileFacilityService < VAOS::SessionService
+      extend Memoist
+
       # Retrieves information about a VA clinic from the VAOS Service.
       #
       # @param station_id [String] the ID of the VA facility where the clinic is located
       # @param clinic_id [String] the ID of the clinic to retrieve
       #
       # @return [OpenStruct] An OpenStruct object containing information about the clinic.
       #
```

Comment:
When I moved the `get_clinic/facility_memoized` methods into this class, I chose to rename them because their distinctive feature isn't really memoization. It's safety. 

The reason we memoize it is to improve performance in rare cases where a facility server is down and the cache hasn't yet been set. The memo prevents us from repeatedly attempting to fetch the same record. We may choose to remove the memoization, but we'll still need the safe version of getting the record.

The only place we use the danger version of the method is when fetching facility show from the controller, because we want it to bomb and let the controller handle the error. 

I'm open to changing these back.

Reply:
I think this will be good at least for now, our approach with the memoization methods initially was one of a few viable approaches we saw, semantics aside. This should be sufficient